src/Apps/Order/Routes/Shipping2/Components/FulfillmentDetailsForm.tsx

```diff
@@ -103,23 +108,35 @@ const FulfillmentDetailsFormLayout = (
     touched,
     handleChange,
     handleBlur,
-    submitForm,
     setFieldValue,
     setValues,
-    isValid,
   } = formikContext
 
   const addressFormMode: AddressFormMode =
     values.fulfillmentType === "SHIP" ? props.shippingMode : "pickup"
 
+  /**
+   * Expose formik context to entire shipping route
+   * via `shippingContext.state.fulfillmentDetailsCtx`
+   */
   useEffect(() => {
-    /**
-     * Pass some key formik bits up to the shipping route
-     * TODO: This could be accomplished with useImperativeHandle(ref, formikContext)
-     */
-    shippingContext.actions.setFormHelpers(formikContext)
+    shippingContext.actions.setFulfillmentDetailsCtx(formikContext)
```

Comment:
We should be more descriptive in the name of this function, just so we can see that its a formik context versus some other type of context:

```tsx
shippingContext.actions.setFulfillmentDetailsFormikContext(formikContext)
```